bylaws.md

```diff
@@ -613,6 +619,19 @@ has a significant consulting relationship with an entity; or that
 person owns at least 1% of the equity or debt, or derivatives 
 thereof, of an entity.
 
+**Section 5.15. Limits on Co-affiliation of Board Members.** No more
+than one quarter (1/4) of the members of the Board of Directors may
+share a common affiliation as defined in Section 5.14. If the number
```

Comment:
@VanL Since 5.14 doesn't include subsideries or parent companies I suggest to strenthen the use of "affiliation" here by including them explicitly to remove any ambiguity around electing affiliates from conglomerates.

Note, I'm not sure about the exact wording here and I trust you to find the right legal form, but I hope my intentions are clear, please let me know if I should elaborate.

```suggestion
share a common affiliation as defined in Section 5.14, including subsideries and parent companies. If the number
```